components/03-sections/03-banner/banner.scss

```diff
@@ -216,35 +218,35 @@
 
 		.carousel-control-prev {
 			width: 8%;
-			height: auto;
+			// height: auto;
 			opacity: 1;
 			z-index: 99;
 		}
 
 		.carousel .carousel-control-next {
```

Comment:
I think `.carousel` is extra on the next and causes the right to be larger than the left since this is already nested under the `.carousel`. 

Also, the left arrow on hover doesn't have a .5 opacity like the right. Once you remove the extra `.carousel` it will fix it, but I like the .5 opacity better.

Reply:
whoops, that was a bad conflict resolution on my end, re-committing

Reply:
I also like 0.5 opacity, will get that too